sources/packages/backend/workflow/test/test-utils/mock/mock.utils.ts

```diff
@@ -56,34 +58,48 @@ export function getNormalizedServiceTaskId(serviceTaskId: string) {
 /**
  * Information to create a mocked worker including the job completed object and/or the
  * messages to be published to unblock the workflow.
- * @param serviceTaskId workflow service task id.
- * @param options mock creation options.
- * - `jobCompleteMock` the object to be returned when the job is completed.
- * If not provided a job complete message will be returned either way.
- * - `jobMessageMocks` optional messages to be sent to the workflow 'message intermediate catch event'.
- * For instance, when the assessment-gateway detects that the partner information is required and it
- * needs to be informed when the partner provided the information.
- * - `subprocesses` one or more workflow subprocess that contains the service
- * task id being mocked. For instance, the 'create-income-request-task' can be execute
- * multiple times (student, partner, parents) but it will happen under different
- * subprocesses.
- * Multiple parameters here would indicate the hierarchy of the subprocesses
- * till the service task id is found (e.g. 'create-income-request-task'). For instance,
- * if the service task id is a child of the SubprocessB and SubprocessB is a child of
- * SubprocessA, this array needs to be populated as [SubprocessA, SubprocessB], which will
- * create the service task unique identifier as SubprocessA_SubprocessB_create_income_request_task.
- * The most common scenario will be one subprocess, for instance, for 'create-income-request-task' under
- * the assessment-gateway workflow:
- * - for student income verification it would be studentIncomeVerificationSubprocess_create_income_request_task
- * - for partner income verification it would be partnerIncomeVerificationSubprocess_create_income_request_task
- * @returns mocked object to be used by a worker.
  */
 export interface WorkerMockedData {
+  /**
```

Comment:
The comments here were moved from the "method style" to the "interface" style, as they should be. No changes were made other than adding the comments to the new property `multiInstanceLoopCounter`.